python/py-pydantic/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pydantic
+version             1.5.1
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
```

Comment:
this package is compiling stuff, check the logs: 
```
/usr/bin/clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -pipe -Os -isysroot/Library/Developer/CommandLineTools/SDKs/MacOSX10.14.sdk -O3 -I/opt/local/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c pydantic/class_validators.c -o build/temp.macosx-10.14-x86_64-3.8/pydantic/class_validators.o
```
So there will be architecture-dependent files and that means you should remove the `supported_archs     noarch` line.